### PR TITLE
Updating Mesos and Marathon metric scripts to support port number config

### PR DESCRIPTION
**Pull request by lusid**

### CODE REVIEW

File: bin/metrics-mesos.rb [hunk 1]
```diff
@@ -33,6 +33,10 @@
 require 'socket'
 require 'json'
 
+# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
+MASTER_DEFAULT_PORT = '5050'
+SLAVE_DEFAULT_PORT = '5051'
```
Comment (tas50) [inline bin/metrics-mesos.rb]: We should be using the default: attribute that options provide in sensu-plugin here.

Comment (lusid) [inline bin/metrics-mesos.rb]: I copied what you already had in check-mesos.rb. If the default ports are 5050 for master nodes and 5051 for slave nodes, then you wouldn't be able to support both unless you did it this way.

Comment (analytically) [inline bin/metrics-mesos.rb]: @mattyjones what's your thoughts here?

Comment (mattyjones) [inline bin/metrics-mesos.rb]: @analytically @lusid  There are two schools of thoughts here. We can leave them defined as the options make no distinction between slave/master so we define them outside.

We could create a new options one for each case but that could be ugly.  As the code is now in the commit is fine for the use case.

**Comment by mattyjones** [comment]
@lusid  Can you update the CHANGELOG and take a look at the tests and we can merge this,

Thanks!

**Comment by analytically** [comment]
bin/metrics-mesos.rb:64:1: C: Trailing whitespace detected.

**Comment by eheydrick** [comment]
Bump for rubocop fixes.

**Pull request merged**